build-data-capturing-gradle-samples/capture-git-diffs/gradle-git-diffs.gradle

```
@@ -21,80 +21,102 @@ if (!buildScanApi) {
     return
 }
 
+def capture = new Capture(gradle.startParameter.offline, gradle.startParameter.continuous, gradle.rootProject.logger, getProviders().gradleProperty('gistToken'), project.name)
```

```suggestion
def capture = new Capture(gradle.startParameter.offline, gradle.startParameter.continuous, gradle.rootProject.logger, providers.gradleProperty('gistToken'), project.name)
```

It seems to me that the original used `rootProject` here, was it intentionally changed?

---

Not sure what you mean by that

---

Ah i mean we changed `rootProject.name` for `project.name`